test/test_vmap.py

```diff
@@ -984,24 +983,6 @@ def f(x, y):
         expected = torch.mv(y, torch.ones(2)).view(3, 1, 1) + x
         self.assertEqual(out, expected)
 
-    def test_tensor_print(self):
-        x = torch.tensor([[3.14]])
-        buf = None
-
-        def foo(x):
-            nonlocal buf
-            buf = repr(x)
-            return x
-
-        vmap(vmap(foo))(x)
-        expected = textwrap.dedent("""\
-                BatchedTensor(lvl=3, bdim=0, value=
-                    BatchedTensor(lvl=2, bdim=0, value=
-                        tensor([[3.1400]])
-                    )
-                )""")
-        self.assertEqual(buf, expected)
```

Comment:
Let's add a test for different bdims here (to make sure we didn't hardcode bdim=0 lol)